pipeline_steps/deploy.groovy

```diff
@@ -55,11 +55,22 @@ def upgrade(String stage_name, String upgrade_script, List env_vars) {
           sh "git reset --hard"
         }
         common.prepareRpcGit()
```

Comment:
If `vm` is set, how does the repo get updated on it? On the initial deployment the repo appears to get copied across, https://github.com/rcbops/rpc-gating/blob/master/pipeline_steps/multi_node_aio_prepare.groovy#L53, but I can't see anything similar here with the upgrade step.